jans-auth-server/server/src/main/java/io/jans/as/server/authorize/ws/rs/AuthorizeRestWebServiceImpl.java

```diff
@@ -602,6 +601,12 @@ private Response requestAuthorization(
                 }
             }
 
+            if (identity.getSessionId().getState() == SessionIdState.AUTHENTICATED
+                    && client != null && Boolean.TRUE.equals(client.getAttributes().getDefaultPromptLogin())
+                    && new Date().getTime() - identity.getSessionId().getAuthenticationTime().getTime() > 200) {
```

Comment:
`new Date().getTime() - identity.getSessionId().getAuthenticationTime().getTime() > 200`
What is the idea of 200ms? Sounds as some case driven by test case.

Reply:
The condition to force re-authentication is when the client attribute defaultPromptLogin is set to True. But we do not whant to stuck forcing re-authentication forever, so I am checking for a small time difference to know whether the session was reccently authenticated then it means it was re-authenticated o authenticated for the first time. Otherwise it is old and requires to re-authenticate.

Reply:
Got it.